### PR TITLE
feat(rgbpp): change apir of asset for rgb++

### PR DESCRIPTION
**Pull request by Daryl-L**
https://github.com/Magickbase/ckb-explorer-public-issues/issues/667

### CODE REVIEW

File: src/services/ExplorerService/fetcher.ts [hunk 1]
```diff
@@ -254,6 +254,11 @@ export const apiFetcher = {
   fetchBitcoinAddresses: (address: string) =>
     requesterV2.get(`bitcoin_addresses/${address}`).then(res => toCamelcase<BitcoinAddresses>(res.data)),
 
+  fetchBitcoinAddressesUDTAccounts: (address: string) =>
```
Comment (Keith-CY) [inline src/services/ExplorerService/fetcher.ts:257]: The name could be more intuitive, such as `fetchUDTAccountsByBtcAddress`, the `address` is singular, so `BitcoinAddresses` should be `BitcoinAddress`

File: src/pages/Address/BTCAddressComp.tsx [hunk 2]
```diff
@@ -18,14 +18,19 @@ enum AssetInfo {
...
   const { data } = useQuery(['bitcoin addresses', address], () =>
     explorerService.api.fetchBitcoinAddresses(address.bitcoinAddressHash || ''),
   )
   const { boundLiveCellsCount, unboundLiveCellsCount } = data || { boundLiveCellsCount: 0, unboundLiveCellsCount: 0 }
 
+  let { data: udtAccounts } = useQuery(['bitcoin address udt accounts', address], async () => {
+    const data = await explorerService.api.fetchBitcoinAddressesUDTAccounts(address.bitcoinAddressHash || '')
+    return data.udtAccounts
+  })
+  udtAccounts = udtAccounts || []
```
Comment (Keith-CY) [inline src/pages/Address/BTCAddressComp.tsx:32]: fallback `[]` can be set in the third parameter `option.initialData`

File: src/services/ExplorerService/fetcher.ts (hunk 1 above)
Comment (Keith-CY) [reply src/services/ExplorerService/fetcher.ts:257]: It's different. `bitcoin_addresses/{address}/udt_accounts` follows RESTful which is semantic. `UDT_Account` is a resource owned by a `bitcoin_address`. `BitcoinAddressesUDTAccounts` is simply a combination of nouns.

**Review by Keith-CY — APPROVED** [review]

**Comment by Keith-CY** [comment]
Is there a page for reviewing the `invalid assets`?

**Comment by FrederLu** [comment]
Since the synchronization service is stopped now, abnormal unbinding data will be removed from the RGB++ tab after manual triggering.At the same time, the abnormal unbinding data appears in `Invalid`.

**Review by FrederLu — APPROVED** [review]

**Pull request merged**